### ISSUE TITLE
FontDialog always return Tahoma

### ISSUE THREAD

**Issue by IncubusRK** [madewokherd/wine-mono#119]
When using FontDialog for select font it always returns "Tahoma" out-dependent on choice
Check with native win32 app and font selection work fine

**Comment by IncubusRK** [comment]
sample test code:

```
FontDialog dlg = new FontDialog();
            if (dlg.ShowDialog() == DialogResult.OK)
            {
                MessageBox.Show("Selected font:"+
                "\r\nFontFamily:\t" + dlg.Font.FontFamily.ToString()+
                "\r\nName:\t\t" + dlg.Font.Name+
                "\r\nSize:\t\t" + dlg.Font.Size.ToString());
            }
```

**Comment by nsivov** [comment]
I suspect this could be about WM_CHOOSEFONT_GETLOGFONT that WinForms is using, they expect LOGFONTW to be filled on return, and we treat it as LOGFONTA.

**Comment by IncubusRK** [comment]
Yes it's look like return GDIPlus.GdipGetGenericFontFamilySansSerif after internal exception or some thing like that

**Comment by nsivov** [comment]
I might be wrong though. Could you upload compiled test binary somewhere so I can test?

**Comment by nsivov** [comment]
I take that back, I see correct name returned from the dialog, but then something weird happens in gdiplus.

**Comment by IncubusRK** [comment]
> I might be wrong though. Could you upload compiled test binary somewhere so I can test?
[WFFontDlg.zip](https://github.com/madewokherd/wine-mono/files/7177022/WFFontDlg.zip)
Test app

**Comment by IncubusRK** [comment]
> I take that back, I see correct name returned from the dialog, but then something weird happens in gdiplus.
I also have some errors in gdiplus when try to create Font https://bugs.winehq.org/show_bug.cgi?id=51753

**Comment by HinTak** [comment]
This actually sound a lot like it is unrelated to wine or mono, but a fontconfig behavior. Fontconfig can be annoying like that.

**Comment by IncubusRK** [comment]
Native call ChooseFont() returns correct font name.
```
CHOOSEFONT cf;
static LOGFONT lf;
memset((void*)&cf, 1, sizeof(cf));
cf.lStructSize = sizeof(cf);
cf.hwndOwner = NULL;
cf.lpLogFont = &lf;
cf.Flags = CF_SCREENFONTS | CF_EFFECTS;
ChooseFont(&cf);
```
